docs/_posts/examples/3400-01-16-point-from-geocoder-result.html

```diff
@@ -2,12 +2,12 @@
 layout: example
 category: example
 title: Set a point after Geocoder result
-description: 'Listen to the <code>geocoder.input</code> event from the <a target="_blank" href="https://www.mapbox.com/mapbox-gl-js/plugins/#mapbox-gl-geocoder">Geocoder plugin</a> and place a point on the coordinate results.'
+description: 'Listen to the <code>geocoder.input</code> event from the <a target="_blank" href="https://github.com/mapbox/mapbox-gl-geocoder">Geocoder plugin</a> and place a point on the coordinate results.'
 tags:
   - controls-and-overlays
 ---
-<script src='https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-geocoder/v{{site.data.plugins.mapbox-gl-geocoder.latest}}/mapbox-gl-geocoder.js'></script>
-<link rel='stylesheet' href='https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-geocoder/v{{site.data.plugins.mapbox-gl-geocoder.latest}}/mapbox-gl-geocoder.css' type='text/css' />
+<script src='https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-geocoder/v2.0.1/mapbox-gl-geocoder.js'></script>
```

Comment:
The plan is for these to be manually updated alongside plugin releases, as they are now.